drivers/wifi/nrf700x/src/wifi_mgmt_scan.c

```diff
@@ -137,10 +137,30 @@ int nrf_wifi_disp_scan_zep(const struct device *dev, struct wifi_scan_params *pa
 		}
 
 		scan_info->scan_params.bands = params->bands;
-		scan_info->scan_params.dwell_time_active = params->dwell_time_active;
-		scan_info->scan_params.dwell_time_passive = params->dwell_time_passive;
 
-		vif_ctx_zep->max_bss_cnt = params->max_bss_cnt;
+		if ((params->dwell_time_active < 5) || (params->dwell_time_active > 1000)) {
```

Comment:
Generic comment: Please create defines for all the magic numbers used (such as `5`, `1000` etc.) maybe in wifi_mgmt.h and then use those defines.

Reply:
Ok. we can create define all these magic numbers. Will it be approved in upstream wifi_mgmt.h?